app/src/main/res/layout/feedback_item.xml

```diff
@@ -10,18 +10,23 @@
   <FrameLayout
     android:layout_width="wrap_content"
     android:layout_height="wrap_content"
-    android:layout_margin="4dp"
-    android:paddingEnd="8dp">
+    android:background="@drawable/content_blue_background"
+    android:layout_marginStart="20dp"
+    android:layout_marginEnd="28dp"
+    android:layout_marginTop="24dp">
 
     <TextView
       android:id="@+id/feedback_text_view"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
-      android:layout_margin="4dp"
-      android:background="@drawable/content_blue_background"
-      android:padding="12dp"
+      android:paddingStart="10dp"
```

Comment:
Not a multiple of 4.

Reply:
Suggest discuss this with @mschanteltc today.

Reply:
+

Reply:
made it 12dp.